wcfsetup/install/files/lib/data/user/group/UserGroup.class.php

```diff
@@ -69,7 +69,7 @@ class UserGroup extends DatabaseObject implements ITitledObject
 
     /**
      * group cache
-     * @var UserGroup[]
+     * @var UserGroup[][]|int[][]
```

Comment:
This should rather be an array shape or completely cleaned up to not require this type in the first place.